python/ray/experimental/workflow/recovery.py

```diff
@@ -129,3 +129,31 @@ def resume_workflow_job(workflow_id: str,
             workflow_context.set_workflow_step_context(None)
 
     return ray.put(reader.load_step_output(r))
+
+
+def get_latest_output(workflow_id: str, store: storage.Storage) -> Any:
```

Comment:
Shouldn't we've already forward the output?

Reply:
this function loads the latest output (or current state) from the storage. It is not directly related to forwarding output.